packages/flutter/lib/src/widgets/page_storage.dart

```diff
@@ -130,6 +130,49 @@ class PageStorageBucket {
 }
 
 /// A widget that establishes a page storage bucket for this widget subtree.
+///
+/// {@tool snippet}
+///
+/// This sample creates a page storage for MyHomePage
```

Comment:
```suggestion
/// This sample creates a page storage for MyHomePage.
```

Could this provide a little bit more context? Such as why we would do that? This example does not seem to show how this can actually be used in practice.

Reply:
This example just illustrates how to use Pagestorage and read and write data into the Pagestorage.